core/src/main/scala/org/apache/spark/h2o/utils/H2OSchemaUtils.scala

```diff
@@ -341,21 +341,18 @@ object H2OSchemaUtils {
     */
   def collectSparseInfo(flatDataFrame: DataFrame, elemMaxSizes: Array[Int]): Array[Boolean] = {
     val vectorIndices = collectVectorLikeTypes(flatDataFrame.schema)
-
-    val sparseInfoForVec = {
-      if (flatDataFrame.isEmpty) {
```

Comment:
@jakubhava The main reason for creating this PR is that [`isEmpty`](https://github.com/apache/spark/blob/master/sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala#L559) was introduced in Spark 2.4. Older versions doesn't support the method.

Reply:
oki, thanks. One would think this method was there from beginning of dataframes :)